examples/0_Simple/vectorSqrt/device.gpr

```diff
@@ -11,9 +11,16 @@ project Device is
 
    for Create_Missing_Dirs use "True";
 
+   for Library_Name use "kernel";
+   for Library_Dir use "lib";
+
+   for Archive_Builder use ("cuda-gcc", "-v");
```

Comment:
Could `-v` be part of `package Builder`?

Reply:
I don't know, this was copy/pasted from another gprfile Quentin wrote. I'll look into whhat using `-v` for `Builder` entails, thanks for the suggestion :)

Reply:
From what I can gather it applies as default flag to Compiler, Binder, and Linker, which seems good for the `-v` flag

Reply:
What would be the right incantation? I tried
```ada
   package Builder is
      for Default_Switches ("Ada") use ("-v");
      -- for Default_Switches ("main") use ("-v");
      -- for Default_Switches ("gnatcuda_wrapper.adb") use ("-v");
      -- for Switches ("Ada") use ("-v");
      -- for Switches ("main") use ("-v");
      -- for Switches ("gnatcuda_wrapper.adb") use ("-v");
   end Builder;
```
And nothing worked.

Reply:
You don't need Archive_Builder anymore - I moved these in the gprbuild database with Vasiliy at the begining of the week. You can remove Archive_Indexer, Archive_Suffix as well as the Object_File_Suffix of the compiler package as well - as long as you have a recent enough toolchain installed. 

I was planning to update all examples with that as soon as I have builds, which should have been today ;-)